kong/pdk/response.lua

```diff
@@ -472,6 +474,28 @@ local function new(self, major_version)
   end
 
 
+  ---
+  -- Redirect uri in the response sent to the client.
+  --
+  -- @function kong.response.redirect
+  -- @phases rewrite, access, header_filter, admin_api
+  -- @tparam string uri The name of the url to be redirect
+  -- @return Nothing; throws an error on invalid input.
+  -- @usage
+  -- kong.response.redirect("https://github.com/kong/kong")
+  function _RESPONSE.redirect(uri)
```

Comment:
Suggest that to add an optional status parameter here as `_RESPONSE.redirect(uri, status?)`, keep the consistency with https://github.com/openresty/lua-nginx-module#ngxredirect.